### ISSUE TITLE
Audit compilability

### ISSUE THREAD

**Issue by cpressey** [catseye/Befunge-93#10]
Several of the C source files, in their header comments, claim they compile under certain compilers. In fact these claims have not been recently tested.

Before the final release of 2.24, but only after everything is merged into [develop-2018-1](https://github.com/catseye/Befunge-93/tree/develop-2018-1), try to compile each C source on the following compilers-system combinations, and note the results more clearly in the header comments.

- Modern gcc on Ubuntu Linux
- Modern gcc on NetBSD
- DICE C on AmigaDOS 1.3
- DJGPP on FreeDOS
- Borland C++ 3.1 (if possible) on FreeDOS

If anyone reading this issue is equipped to compile C programs with a Metrowerks compiler (I guess this means CodeWarrior) on a Mac, they're free to see if those claims still hold; but I won't bank on this and will simply state that at one point it appears that they compiled under that combination.

**Comment by cpressey** [comment]
This is done. If I was being uber-perfectionist I would check these all again after [this change to bef.c](https://github.com/catseye/Befunge-93/pull/12/commits/519bab9b19b593489a374ec15932c4fef9be16d5#diff-fc555da03e1d6d9bd6d22e01fa8d7ecf) but seriously, no, that's OK. I just spot-checked the Amiga build and it's still okay. So, closing this.
